From 81d1be927e0db5eb838d85c2a55c9ffbaafa8792 Mon Sep 17 00:00:00 2001
From: Kay Sievers <kay@vrfy.org>
Date: Fri, 31 Jul 2015 15:33:49 +0200
Subject: [PATCH 25/47] timesyncd: remove RLIMIT_NPROC

NSS plugins might create additional threads. Remove the limit, we cannot
really make any assumptions here.
---
 src/timesync/timesyncd.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/timesync/timesyncd.c b/src/timesync/timesyncd.c
index b030206..7b4178c 100644
--- a/src/timesync/timesyncd.c
+++ b/src/timesync/timesyncd.c
@@ -113,10 +113,6 @@ int main(int argc, char *argv[]) {
         if (r < 0)
                 goto finish;
 
-        /* We need one process for ourselves, plus one thread for the asynchronous resolver */
-        if (setrlimit(RLIMIT_NPROC, &RLIMIT_MAKE_CONST(2)) < 0)
-                log_warning_errno(errno, "Failed to lower RLIMIT_NPROC to 2: %m");
-
         assert_se(sigprocmask_many(SIG_BLOCK, NULL, SIGTERM, SIGINT, -1) >= 0);
 
         r = manager_new(&m);
-- 
2.5.0

